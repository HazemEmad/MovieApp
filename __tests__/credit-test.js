/**
 * @format
 */

 import React from 'react';
 import renderer from 'react-test-renderer';
 import Credit from '../src/components/credit';
 
 test('renders correctly', () => {
   const tree = renderer.create(<Credit />).toJSON();
   expect(tree).toMatchSnapshot();
 });