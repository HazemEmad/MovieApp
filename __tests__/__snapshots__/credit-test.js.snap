// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginRight": 5,
        "paddingBottom": 7,
        "paddingLeft": 7,
        "paddingRight": 7,
        "paddingTop": 7,
      },
    ]
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/default-user.png",
      }
    }
    style={
      Array [
        Object {
          "backgroundColor": "#e2e2e2",
          "borderRadius": 35,
          "height": 70,
          "resizeMode": "cover",
          "width": 70,
        },
      ]
    }
  />
  <Text
    style={
      Array [
        Object {
          "color": "#001321",
        },
      ]
    }
  />
</View>
`;
